app/helpers/application_helper.rb

```diff
@@ -4,6 +4,7 @@ module ApplicationHelper
   include_concern 'FormTags'
   include Sandbox
   include CompressedIds
+  include TextualHelper
```

Comment:
:pensive:

I obviously got too distracted un-DRYing the rest of the not-quite-identical instances